lib/rspec/core/notifications.rb

```diff
@@ -414,6 +414,24 @@ def duplicate_rerun_locations
           end
         end
       end
+
+      def rspec_shell_command
+        @rspec_shell_command ||= begin
+          if spring_rspec?
```

Comment:
This doesn't reflect its output, if a binstub is present we could default to it, else use the global version.